kidviz/models.py

```diff
@@ -173,6 +173,18 @@ class Observation(TimeStampedModel, OwnerMixin):
 
     observation_date = models.DateField(default=now)
 
+    @property
+    def allowed_student(self):
```

Comment:
Nitpick: since we're returning a list of multiple students, I think the function name should be plural